go.mod

```diff
@@ -45,6 +46,7 @@ require (
 	github.com/gorilla/mux v1.8.0
 	github.com/gorilla/websocket v1.5.0
 	github.com/gosimple/slug v1.13.1
+	github.com/gosuri/uitable v0.0.4
```

Comment:
Not sure I understand the value of adding this library to the project. I quickly looked at their codebase and it is mainly syntax sugar. Can you pls try to avoid adding this dependency and implement the CLI following the standard that we use in the rest of the project? If you think that there is a huge benefit that maybe I am not able to see please let me know. Otherwise just implement the same using simple `fmt.Printf()`